src/cloud/components/RateLimitAlert.tsx

```diff
@@ -110,6 +116,10 @@ const RateLimitAlert: FC<Props> = ({alertOnly, className, location}) => {
     ? IconFont.AlertTriangle
     : IconFont.Cloud
 
+  if (isFlagEnabled('multiOrg') && !multiOrgFlagForceShow && !alertOnly) {
```

Comment:
if the multiOrg flag is ON, we're hiding the component everywhere unless multiOrgFlagForceShow prop is passed as true.

I checked that the RateLimitAlert is only used in page headers except on dashboard page. Where it's configured to show as an alert.

Reply:
What does `!alertOnly` do?

Reply:
if the component is configured to show as an alert, (using the alertOnly prop), then we don't override (don't return null, let it render the alert)

Reply:
That's the one edge case.

Reply:
What's interesting to me is that if `alertOnly` is `true`, it looks like nothing happens - the component doesn't do anything, since it only has a case for when `alertOnly` is false. (Or, regardless of whether it's true or false, the cardinality conditional can be triggered.) This is kind of an unrelated aside - this just looks like a component that might need some other tweaks later, since `alertOnly === true` doesn't affirmatively do anything.

Reply:
Oh ha! i didn't realize that. That is true, it's checking if it's false.

Reply:
Short update - some revisions incoming here, to make sure that notifications don't duplicate for the copy of this on dashboards.